mne/viz/_3d.py

```diff
@@ -990,6 +1078,225 @@ def time_label(t):
+    time_label : str | callable | None
+        Format of the time label (a format string, a function that maps
+        floating point time values to strings, or None for no label). The
+        default is ``time=%0.3f s``.
+    transparent : bool
+        Whether to use a linear transparency between fmin and fmid. Defaults to
+        True.
+    vector_alpha : float
+        Alpha value to apply globally to the vector glyphs. Defaults to 1.
+    brain_alpha : float
+        Alpha value to apply globally to the surface meshes. Defaults to 0.4.
+    scale_factor : float | None
+        Scaling factor for the vector glyphs. By default, an attempt is made to
+        automatically determine a sane value.
+    subjects_dir : str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+    figure : instance of mayavi.core.scene.Scene | list | int | None
+        If None, a new figure will be created. If multiple views or a
+        split view is requested, this must be a list of the appropriate
+        length. If int is provided it will be used to identify the Mayavi
+        figure by it's id or create a new figure with the given id.
+    views : str | list
+        View to use. See surfer.Brain().
+    colorbar : bool
+        If True, display colorbar on scene.
+    clim : str | dict
+        Colorbar properties specification. If 'auto', set clim automatically
+        based on data percentiles. If dict, should contain:
+
+            ``kind`` : str
+                Flag to specify type of limits. 'value' or 'percent'.
+            ``lims`` : list | np.ndarray | tuple of float, 3 elements
+                Note: Only use this if 'colormap' is not 'mne'.
+                Left, middle, and right bound for colormap.
+            ``pos_lims`` : list | np.ndarray | tuple of float, 3 elements
+                Note: Only use this if 'colormap' is 'mne'.
+                Left, middle, and right bound for colormap. Positive values
+                will be mirrored directly across zero during colormap
+                construction to obtain negative control points.
+
+    size : float or pair of floats
+        The size of the window, in pixels. can be one number to specify
+        a square window, or the (width, height) of a rectangular window.
+    background : matplotlib color
+        Color of the background of the display window.
+    foreground : matplotlib color
+        Color of the foreground of the display window.
+
+    Returns
+    -------
+    brain : Brain
+        A instance of surfer.viz.Brain from PySurfer.
+
+    Note
+    ----
+    PySurfer currently needs the SUBJECTS_DIR environment variable, which will
+    automatically be set by this function. Plotting multiple SourceEstimates
+    with different values for subjects_dir will cause PySurfer to use the wrong
+    FreeSurfer surfaces when using methods of the returned Brain object. It is
```

Comment:
I actually don't think this is true anymore... if you have time to check and fix the docstring please do, if not we can also do it in a follow-up PR